themes/default/content/templates/serverless-application/azure/index.md

```diff
@@ -0,0 +1,148 @@
+---
+title: "Azure Serverless Application"
+layout: template
+
+# Make sure this is description accurate for this template.
+meta_desc: The Azure Serverless Template makes it easy to deploy a serverless application on Azure with Pulumi, Azure Functions, and Azure Blob Storage.
+
+# Appears on the cards on template-overview pages.
+card_desc: Deploy a serverless application on Azure with Pulumi, Azure Functions, and Azure Blob Storage.
+
+# Used for generating language-specific links to templates on GitHub. (Example: `static-website-aws`)
+template:
+    prefix: serverless-azure
+
+# Used for generating links to sibling templates in the right-hand nav. Slug is this template's parent directory.
+cloud:
+  name: Azure
+  slug: azure
+
+# Be sure to replace this image. Figma source file:
+# https://www.figma.com/file/lGrSpwbGGmbixEuewMbtkh/Template-Architecture-Diagrams?node-id=15%3A196
+meta_image: meta.png
+
+# The content below is meant help you get started and to serve as a guide to work by. Feel free to adjust it needed for your template.
+---
+
+The Azure Serverless Application template deploys a serverless application on Azure with Pulumi. It deploys an [Azure Storage Blob account]({{< relref "/registry/packages/azure-native/api-docs/storage/storageaccount" >}}) that's configured with a [container for static website hosting]({{< relref "/registry/packages/azure-native/api-docs/storage/storageaccountstaticwebsite" >}}) and an [Azure function]({{< relref "/registry/packages/azure-native/api-docs/web/webappfunction" >}}) that runs the business logic. The application uses an [App Service plan]({{< relref "/registry/packages/azure-native/api-docs/web/appserviceplan" >}}) by default. The template ships with a placeholder website that displays the current time to give you a working Pulumi project out of the box that you can customize easily and extend to suit your needs.
```

Comment:
I thought we were going with this wording about deploying the IaC project. If so, we should update the AWS serverless one as well

> The Container Service template creates an infrastructure as code project in your favorite language that deploys a container service to AWS.

Reply:
> The AWS Serverless Application template deploys a serverless website to AWS that displays the current time. It deploys an [Amazon S3 bucket](https://www.pulumi.com/registry/packages/aws/api-docs/s3/bucket/) for hosting a static website, deploys an [AWS Lambda function](https://www.pulumi.com/registry/packages/aws/api-docs/lambda/function/) that runs the business logic, and an [Amazon API Gateway REST API](https://www.pulumi.com/registry/packages/aws/api-docs/apigateway/restapi/) that routes requests to HTML content and the Lambda function. The template ships with a placeholder website that displays the current time to give you a working Pulumi project out of the box that you can customize easily and extend to suit your needs.

The AWS one also seems more specific about what is the application being deployed and how the different components work together, more specifically how the blog account and the storage container are doing (i think its holding the static components).

Reply:
@cnunciato could you update the helper with the new boilerplate intro description language per above?

Reply:
i updated the first sentence description. i don't see a major difference between how the aws and azure ones have been written, so i've left it as is